### ISSUE TITLE
GtG starts and closes immediately

### ISSUE THREAD

**Issue by SPichelmann** [getting-things-gnome/gtg#706]
Hi everyone
I removed all subtasks and suddenly gtg closed. Now whenever I try to start it, it immediately closes again. Using one of the auto generated xml backups did not help.

Console input output:
flatpak run org.gnome.GTG
GNOME keyring not found, passwords will be not stored after restarting GTG

**Comment by ebousse** [comment]
Same problem here, currently GTG crashes as soon as it starts.

Here is what I get as console output:
```
$ flatpak run org.gnome.GTG -d
GNOME keyring not found, passwords will be not stored after restarting GTG
2021-09-10 12:48:59,054 - DEBUG - application:set_logging:587 - Debug output enabled.
Gtk-Message: 12:48:59.091: Failed to load module "canberra-gtk-module"
Gtk-Message: 12:48:59.091: Failed to load module "pk-gtk-module"
2021-09-10 12:48:59,092 - DEBUG - __init__:__init__:60 - Backends found: ['backend_localfile', 'backend_signals']
2021-09-10 12:48:59,127 - DEBUG - xml:open_file:233 - Opening file /home/bousse-e/.var/app/org.gnome.GTG/data/gtg/gtg_data.xml
2021-09-10 12:48:59,142 - DEBUG - task:add_child:687 - adding child 935f2012-18bc-4ffd-9376-8c1e2e7cd3aa to task a1b4d57a-6eac-4d87-b7b9-6a48bcb58f47
2021-09-10 12:48:59,143 - DEBUG - task:add_child:687 - adding child 27f41bc1-b26e-4a09-abfb-587c2fa2ae46 to task a1b4d57a-6eac-4d87-b7b9-6a48bcb58f47
2021-09-10 12:48:59,143 - DEBUG - task:add_child:687 - adding child 9420b358-e763-4575-8fc9-570061504616 to task a1b4d57a-6eac-4d87-b7b9-6a48bcb58f47
2021-09-10 12:48:59,143 - DEBUG - task:add_child:687 - adding child 9b367ad6-46c8-47ce-98c1-c5fa80dee1b9 to task a1b4d57a-6eac-4d87-b7b9-6a48bcb58f47
2021-09-10 12:48:59,143 - DEBUG - task:add_child:687 - adding child add8b7db-6908-4cc7-a2cf-c01365330749 to task a1b4d57a-6eac-4d87-b7b9-6a48bcb58f47
2021-09-10 12:48:59,143 - DEBUG - task:add_child:687 - adding child 6fa64220-d454-4aae-b6de-40e39574f824 to task a1b4d57a-6eac-4d87-b7b9-6a48bcb58f47
2021-09-10 12:48:59,143 - DEBUG - task:add_child:687 - adding child fc9d4419-ec3d-40e8-9f57-b0534b4a3169 to task a1b4d57a-6eac-4d87-b7b9-6a48bcb58f47
2021-09-10 12:48:59,143 - DEBUG - task:add_child:687 - adding child 08fbe7c0-98f4-4e3a-a1ad-d104361717dd to task a1b4d57a-6eac-4d87-b7b9-6a48bcb58f47
2021-09-10 12:48:59,151 - DEBUG - task:add_child:687 - adding child 8a61f962-2d86-4e2d-bd00-4e90ddb2c774 to task 762c2207-1b4c-4402-a7a1-3fb064a9624c
2021-09-10 12:48:59,151 - DEBUG - task:add_child:687 - adding child 0e7622d8-ab3e-4a72-8363-7ed8eab6c60d to task 762c2207-1b4c-4402-a7a1-3fb064a9624c
2021-09-10 12:48:59,151 - DEBUG - task:add_child:687 - adding child 44dcbabd-4c1f-48ad-b430-072e3bdb2f31 to task 762c2207-1b4c-4402-a7a1-3fb064a9624c
2021-09-10 12:48:59,151 - DEBUG - task:add_child:687 - adding child 499fe837-5fb2-4a06-bbee-fbdae4fa37a5 to task 762c2207-1b4c-4402-a7a1-3fb064a9624c
2021-09-10 12:48:59,151 - DEBUG - task:add_child:687 - adding child 9b1b0e97-43a0-45cc-ab52-442083f744b7 to task 762c2207-1b4c-4402-a7a1-3fb064a9624c
2021-09-10 12:48:59,152 - DEBUG - task:add_child:687 - adding child c0356638-57d4-446a-bb55-b565708213e5 to task 762c2207-1b4c-4402-a7a1-3fb064a9624c
2021-09-10 12:48:59,170 - DEBUG - task:add_child:687 - adding child bca98e0b-cf01-48e4-901d-13db27171c21 to task 416a7476-fcca-43d0-a52b-2029f6d535f6
2021-09-10 12:48:59,170 - DEBUG - task:add_child:687 - adding child 5f8cf05b-9625-4d98-a6b8-d9302e09de01 to task 416a7476-fcca-43d0-a52b-2029f6d535f6
2021-09-10 12:48:59,170 - DEBUG - task:add_child:687 - adding child 4abd4cf2-e3ad-4823-93b9-d06876d46328 to task 416a7476-fcca-43d0-a52b-2029f6d535f6
2021-09-10 12:48:59,176 - DEBUG - task:add_child:687 - adding child 91f5b6ab-68cc-4444-b765-d867b37849fc to task c679879d-aa22-4cb7-b7f4-135475ca76ff
2021-09-10 12:48:59,176 - DEBUG - task:add_child:687 - adding child 049ce53c-8a8b-41c9-a80b-ff758f68f46f to task c679879d-aa22-4cb7-b7f4-135475ca76ff
2021-09-10 12:48:59,176 - DEBUG - task:add_child:687 - adding child b85dbf35-5686-43ac-ae3a-916fd291c59c to task c679879d-aa22-4cb7-b7f4-135475ca76ff
2021-09-10 12:48:59,183 - DEBUG - task:add_child:687 - adding child cb800b40-d608-48b1-8e44-d808a0ed66ef to task 481d00fa-2635-4de2-88ff-9bae39b24e11
2021-09-10 12:48:59,183 - DEBUG - task:add_child:687 - adding child 839d4a69-402c-463e-86e7-7f361f14bf79 to task 481d00fa-2635-4de2-88ff-9bae39b24e11
2021-09-10 12:48:59,186 - DEBUG - task:add_child:687 - adding child a429cc22-d8bf-4e7f-bc74-1227c82b6187 to task ada7be4c-6974-4c55-a4fd-8b1435f17eb5
2021-09-10 12:48:59,186 - DEBUG - task:add_child:687 - adding child fffc198b-b85b-4af5-97fe-353f1d47a1eb to task ada7be4c-6974-4c55-a4fd-8b1435f17eb5
2021-09-10 12:48:59,186 - DEBUG - task:add_child:687 - adding child 0c005992-71f7-4de7-b2c8-236e50893e0a to task ada7be4c-6974-4c55-a4fd-8b1435f17eb5
2021-09-10 12:48:59,186 - DEBUG - task:add_child:687 - adding child 24fe58d1-75d8-4c4e-a99f-74f1700e4d29 to task ada7be4c-6974-4c55-a4fd-8b1435f17eb5
2021-09-10 12:48:59,186 - DEBUG - task:add_child:687 - adding child cb344559-3dbb-4895-a3bf-c866ccd4b106 to task ada7be4c-6974-4c55-a4fd-8b1435f17eb5
2021-09-10 12:48:59,186 - DEBUG - task:add_child:687 - adding child 91beff66-33fb-449c-b8c5-065016df7d0a to task ada7be4c-6974-4c55-a4fd-8b1435f17eb5
2021-09-10 12:48:59,186 - DEBUG - task:add_child:687 - adding child 1362f059-7411-4bf8-b3c9-cfd5e5d90fdd to task ada7be4c-6974-4c55-a4fd-8b1435f17eb5
2021-09-10 12:48:59,186 - DEBUG - task:add_child:687 - adding child 8b41069d-c4d4-4c99-991c-b67ef085f681 to task ada7be4c-6974-4c55-a4fd-8b1435f17eb5
2021-09-10 12:48:59,186 - DEBUG - task:add_child:687 - adding child 8eb68259-83cf-4839-8d0c-490555178952 to task ada7be4c-6974-4c55-a4fd-8b1435f17eb5
2021-09-10 12:48:59,187 - DEBUG - task:add_child:687 - adding child 26516db4-5534-44ea-89b7-96c40c7496f0 to task ada7be4c-6974-4c55-a4fd-8b1435f17eb5
2021-09-10 12:48:59,187 - DEBUG - task:add_child:687 - adding child 5794f7f3-39e3-4f67-8905-b4d4dc5be1aa to task ada7be4c-6974-4c55-a4fd-8b1435f17eb5
2021-09-10 12:48:59,187 - DEBUG - task:add_child:687 - adding child f444245f-f1f5-4a83-b9f7-78b32bb4bb4f to task ada7be4c-6974-4c55-a4fd-8b1435f17eb5
2021-09-10 12:48:59,187 - DEBUG - task:add_child:687 - adding child 2859f2de-8946-45c1-9b3d-5e2750fa558f to task ada7be4c-6974-4c55-a4fd-8b1435f17eb5
2021-09-10 12:48:59,187 - DEBUG - task:add_child:687 - adding child ceef32cc-3f7c-44db-8d41-cfe295e149f4 to task ada7be4c-6974-4c55-a4fd-8b1435f17eb5
2021-09-10 12:48:59,211 - DEBUG - task:add_child:687 - adding child c549b933-ae30-452e-b337-ac98fb6043f8 to task b13b943b-6f9a-4f84-9d59-a0c3bf15476d
2021-09-10 12:48:59,293 - DEBUG - task:add_child:687 - adding child 1501f977-f3e6-43e6-9da9-578297c896d4 to task 070e888f-7239-452e-82ef-48db6e8bb671
2021-09-10 12:48:59,332 - DEBUG - task:add_child:687 - adding child 5e446157-f1b1-463e-a54a-bd02ea453c63 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,332 - DEBUG - task:add_child:687 - adding child ea907fdc-3618-4e5c-b4eb-e0a4a51ff9f4 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,332 - DEBUG - task:add_child:687 - adding child 94336cec-c93f-4be7-b278-2948fa4fae9b to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,332 - DEBUG - task:add_child:687 - adding child 276abfa3-2c7e-443b-aeb7-caba07e8c208 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,332 - DEBUG - task:add_child:687 - adding child 0f543b5a-7ed1-4ec8-8cee-89f99b4d88a7 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,333 - DEBUG - task:add_child:687 - adding child 1c3f2127-4b38-49f6-ab5e-93be141be634 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,333 - DEBUG - task:add_child:687 - adding child b7281470-80aa-4c48-80c3-3e6180eca1d0 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,333 - DEBUG - task:add_child:687 - adding child 34a971ea-e8cd-4fc0-908f-fa2b0c8d1be7 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,333 - DEBUG - task:add_child:687 - adding child 74269d6f-0f3e-42bc-85d2-40623df63689 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,333 - DEBUG - task:add_child:687 - adding child 9dd6ffc3-58b6-444e-a429-f7c732685952 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,333 - DEBUG - task:add_child:687 - adding child b60e7c44-c72f-4411-98c5-651f587e451c to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,333 - DEBUG - task:add_child:687 - adding child 10ae18fe-cf3a-4d2f-8f47-bd5fb40b6aac to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,333 - DEBUG - task:add_child:687 - adding child 96babaa7-cd68-4362-b0fe-8883faaca6e7 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,333 - DEBUG - task:add_child:687 - adding child e46e4944-16ad-402e-b3d0-9963879da703 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,333 - DEBUG - task:add_child:687 - adding child 0ae3dc41-0361-47fb-ae72-5d8aac7355ac to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,333 - DEBUG - task:add_child:687 - adding child 30964030-198a-4d17-92eb-3ec0074e2829 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,333 - DEBUG - task:add_child:687 - adding child 15f3b519-fe59-42b2-bd77-119db7615547 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,333 - DEBUG - task:add_child:687 - adding child 1398406f-77c4-4a3c-958f-122cc1137c08 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,333 - DEBUG - task:add_child:687 - adding child 0deebd97-ba2d-4480-91b0-053fe8d550b5 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,333 - DEBUG - task:add_child:687 - adding child 210f473f-acfd-4cbe-b208-b8bead1fd1d0 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,333 - DEBUG - task:add_child:687 - adding child 7d9711c2-cf64-4017-bd1b-42fc7916fc23 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,333 - DEBUG - task:add_child:687 - adding child 4d5a1f25-d314-411b-87f9-b005d08f6aba to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,334 - DEBUG - task:add_child:687 - adding child 4874b9d4-b071-4deb-b182-4f229cc1c566 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,334 - DEBUG - task:add_child:687 - adding child 94ed2057-d140-4136-a031-94126cd433b5 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,334 - DEBUG - task:add_child:687 - adding child 5fd5261e-7ac0-449c-9458-a5400a590e7b to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,334 - DEBUG - task:add_child:687 - adding child 857b8bbe-2346-40dc-a4dd-2c1a1cad7b20 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,334 - DEBUG - task:add_child:687 - adding child 59f57420-dad4-4066-a613-0759e2837fa5 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,334 - DEBUG - task:add_child:687 - adding child f061cda6-3baa-4ad5-b4fc-40b74015d1cc to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,334 - DEBUG - task:add_child:687 - adding child 8e17b1dc-cd51-447d-ab2a-9a5e7ae0ca34 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,334 - DEBUG - task:add_child:687 - adding child ddc2168f-687e-498b-a24f-b36abf0eb4cc to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,334 - DEBUG - task:add_child:687 - adding child 52644bbe-c53f-471e-ae1a-35658ea56968 to task 8b41069d-c4d4-4c99-991c-b67ef085f681
2021-09-10 12:48:59,336 - DEBUG - task:add_child:687 - adding child 13bf6900-8362-482e-8799-86ee14bb9ce6 to task 8eb68259-83cf-4839-8d0c-490555178952
2021-09-10 12:48:59,337 - DEBUG - task:add_child:687 - adding child 8b21d122-9856-41ed-8452-05fc6ec7e0ff to task 8eb68259-83cf-4839-8d0c-490555178952
2021-09-10 12:48:59,337 - DEBUG - task:add_child:687 - adding child 01c2f90f-7e0c-41b1-97e1-89db4255bb65 to task 8eb68259-83cf-4839-8d0c-490555178952
2021-09-10 12:48:59,337 - DEBUG - task:add_child:687 - adding child b23addb5-04a7-4890-bc72-946de15bbd02 to task 8eb68259-83cf-4839-8d0c-490555178952
2021-09-10 12:48:59,337 - DEBUG - task:add_child:687 - adding child f235c3eb-ea26-4442-a62d-4d8bc62b1e9e to task 8eb68259-83cf-4839-8d0c-490555178952
2021-09-10 12:48:59,337 - DEBUG - task:add_child:687 - adding child 559a2cc4-6fa3-40d1-95d2-39ba0897021c to task 8eb68259-83cf-4839-8d0c-490555178952
2021-09-10 12:48:59,337 - DEBUG - task:add_child:687 - adding child 4fb2ef39-b606-4d1b-95ea-1d0a7a6b70d1 to task 8eb68259-83cf-4839-8d0c-490555178952
2021-09-10 12:48:59,337 - DEBUG - task:add_child:687 - adding child fda7b835-eb8a-4bb0-83b3-919fd3c7260f to task 8eb68259-83cf-4839-8d0c-490555178952
2021-09-10 12:48:59,337 - DEBUG - task:add_child:687 - adding child 8bece0a0-5355-4ec1-b903-88e7b5f5fbf5 to task 8eb68259-83cf-4839-8d0c-490555178952
2021-09-10 12:48:59,339 - DEBUG - task:add_child:687 - adding child f7983b8c-f2c6-4141-890a-6dcb23af55f7 to task 26516db4-5534-44ea-89b7-96c40c7496f0
2021-09-10 12:48:59,339 - DEBUG - task:add_child:687 - adding child 815e59ca-b061-4d20-82c0-a8626bde0454 to task 26516db4-5534-44ea-89b7-96c40c7496f0
2021-09-10 12:48:59,339 - DEBUG - task:add_child:687 - adding child 83ccfcca-f781-49f8-9f4a-1832862ebd6b to task 26516db4-5534-44ea-89b7-96c40c7496f0
2021-09-10 12:48:59,339 - DEBUG - task:add_child:687 - adding child 1b130189-1afc-4648-b4b8-92040b103e4f to task 26516db4-5534-44ea-89b7-96c40c7496f0
2021-09-10 12:48:59,339 - DEBUG - task:add_child:687 - adding child 54ad4a3f-1451-43b2-81c3-ac32baf91479 to task 26516db4-5534-44ea-89b7-96c40c7496f0
2021-09-10 12:48:59,339 - DEBUG - task:add_child:687 - adding child eaedf801-db01-4b76-aa02-dd9f5e5f960d to task 26516db4-5534-44ea-89b7-96c40c7496f0
2021-09-10 12:48:59,339 - DEBUG - task:add_child:687 - adding child 7968b150-1e2f-480c-8124-d35f76c8fb3c to task 26516db4-5534-44ea-89b7-96c40c7496f0
2021-09-10 12:48:59,339 - DEBUG - task:add_child:687 - adding child 80dac402-2f92-4e82-a343-acb971978456 to task 26516db4-5534-44ea-89b7-96c40c7496f0
2021-09-10 12:48:59,340 - DEBUG - task:add_child:687 - adding child 70db1c8b-330e-471e-8adb-599479844bff to task 26516db4-5534-44ea-89b7-96c40c7496f0
2021-09-10 12:48:59,340 - DEBUG - task:add_child:687 - adding child 8c4a0b4d-c8e8-40a2-8c85-bf8fafeb4f74 to task 26516db4-5534-44ea-89b7-96c40c7496f0
2021-09-10 12:48:59,340 - DEBUG - task:add_child:687 - adding child 04e4cc74-0323-4d68-8c73-656f2a8ab760 to task 26516db4-5534-44ea-89b7-96c40c7496f0
2021-09-10 12:48:59,342 - DEBUG - task:add_child:687 - adding child ffcdb864-2727-48da-b88b-6ce40c142525 to task 5794f7f3-39e3-4f67-8905-b4d4dc5be1aa
2021-09-10 12:48:59,342 - DEBUG - task:add_child:687 - adding child b4b9325b-f628-43fd-a6d1-894085ed3ffd to task 5794f7f3-39e3-4f67-8905-b4d4dc5be1aa
2021-09-10 12:48:59,342 - DEBUG - task:add_child:687 - adding child d283c5e6-268e-4d3c-a56f-5265be50bd19 to task 5794f7f3-39e3-4f67-8905-b4d4dc5be1aa
2021-09-10 12:49:00,697 - DEBUG - application:do_activate:126 - Application activation finished
2021-09-10 12:49:00,845 - DEBUG - main_window:on_task_treeview_button_press_event:939 - Received button event #1 at 643,240
2021-09-10 12:49:00,887 - DEBUG - taskview:process:175 - Processing text buffer after 250ms
2021-09-10 12:49:00,895 - DEBUG - taskview:process:225 - Processed in 8.12ms
2021-09-10 12:49:00,931 - DEBUG - taskview:insert_existing_subtask:816 - Task 835410ae-30d1-441d-9c1c-6092c8682ec5 not found
2021-09-10 12:49:00,931 - DEBUG - taskview:insert_existing_subtask:816 - Task e76ceef7-3332-4570-a3bf-df9d647dd950 not found
2021-09-10 12:49:00,931 - DEBUG - taskview:insert_existing_subtask:816 - Task ec6fabb2-6c1c-475e-8c11-963b2d544176 not found
2021-09-10 12:49:00,931 - DEBUG - taskview:insert_existing_subtask:816 - Task 5cfe856e-1a30-4710-9f4b-790d935ef2ba not found
2021-09-10 12:49:00,931 - DEBUG - taskview:insert_existing_subtask:816 - Task 9d084274-67d6-4382-b44a-dc9c3ab3e121 not found
2021-09-10 12:49:00,932 - DEBUG - taskview:insert_existing_subtask:816 - Task 6612e508-1a2d-4ebc-80ad-5b3c28fb5d5f not found
2021-09-10 12:49:00,932 - DEBUG - taskview:insert_existing_subtask:816 - Task c55afc0a-9ee7-4e76-9550-c8e55dd07205 not found
2021-09-10 12:49:00,932 - DEBUG - taskview:insert_existing_subtask:816 - Task e97ee031-bc51-47f4-a3af-d5769f1195b8 not found
2021-09-10 12:49:00,932 - DEBUG - taskview:insert_existing_subtask:816 - Task e1d8434b-ea3e-4d26-a87d-c602a28a166c not found
2021-09-10 12:49:00,932 - DEBUG - taskview:insert_existing_subtask:816 - Task ea1d499b-68c4-45a2-a0fa-a7a8bbe0c02e not found
2021-09-10 12:49:00,932 - DEBUG - taskview:insert_existing_subtask:816 - Task 86a8d9f0-e9cf-468a-a9d3-1ceb16bee63b not found
2021-09-10 12:49:00,932 - DEBUG - taskview:insert_existing_subtask:816 - Task f74c70a4-3201-454e-9231-e0e24c6b5813 not found
2021-09-10 12:49:00,932 - DEBUG - taskview:insert_existing_subtask:816 - Task 4d6408de-90df-4c86-bc1a-a38f9db6b56a not found
2021-09-10 12:49:00,932 - DEBUG - taskview:insert_existing_subtask:816 - Task ff96cb1e-55b1-4379-98b2-807c6c87390d not found
2021-09-10 12:49:00,932 - DEBUG - taskview:insert_existing_subtask:816 - Task 8b97bc9b-823e-400a-b4fe-499fa10b6150 not found
2021-09-10 12:49:00,932 - DEBUG - taskview:insert_existing_subtask:816 - Task 976dffef-1953-4f3b-97d8-e21862df76b0 not found
2021-09-10 12:49:00,932 - DEBUG - taskview:insert_existing_subtask:816 - Task 61c30d23-9e64-4983-b84c-4597bb85bc25 not found
2021-09-10 12:49:00,933 - DEBUG - taskview:insert_existing_subtask:816 - Task ca6f3458-c136-4faf-80e6-4c8047adeb2c not found
2021-09-10 12:49:00,933 - DEBUG - taskview:insert_existing_subtask:816 - Task 2610c443-9e03-4116-b878-8ba911908f98 not found
2021-09-10 12:49:00,933 - DEBUG - taskview:insert_existing_subtask:816 - Task 85266b27-d3a1-48c0-8b6f-165149793bae not found
2021-09-10 12:49:00,933 - DEBUG - taskview:insert_existing_subtask:816 - Task 9951aa65-1eef-4d79-959d-9309c673c41b not found
2021-09-10 12:49:00,933 - DEBUG - taskview:insert_existing_subtask:816 - Task 00e5a9d7-459f-41f9-a17b-bb54b10fe90c not found
2021-09-10 12:49:00,933 - DEBUG - taskview:insert_existing_subtask:816 - Task 45f76b46-2d59-4a09-9990-13d38b7a767f not found
2021-09-10 12:49:00,933 - DEBUG - taskview:insert_existing_subtask:816 - Task fa939f62-df20-481f-ac77-13d8fa13eec3 not found
2021-09-10 12:49:00,933 - DEBUG - taskview:insert_existing_subtask:816 - Task 980a1d0f-d067-4d0e-92db-de6a3756d13f not found
2021-09-10 12:49:00,933 - DEBUG - taskview:insert_existing_subtask:816 - Task aa98d6a2-ef31-4aa9-a66d-0f101c00d32d not found
2021-09-10 12:49:00,933 - DEBUG - taskview:insert_existing_subtask:816 - Task 9e7d197d-94a9-4928-8c36-c49388489c4f not found
2021-09-10 12:49:00,933 - DEBUG - taskview:insert_existing_subtask:816 - Task 3083c580-7e83-4e58-ad97-f904a0681b2e not found
2021-09-10 12:49:00,933 - DEBUG - taskview:insert_existing_subtask:816 - Task d0ab9c74-b993-4d54-be48-82168487ad44 not found
2021-09-10 12:49:00,933 - DEBUG - taskview:insert_existing_subtask:816 - Task bda865e1-9b4c-408d-9de0-272f709f4bac not found
2021-09-10 12:49:00,933 - DEBUG - taskview:insert_existing_subtask:816 - Task 0aa98e39-0f61-4a16-a0eb-85046134c5e5 not found
2021-09-10 12:49:00,934 - DEBUG - taskview:insert_existing_subtask:816 - Task 321bdf31-5a58-4b8f-93f0-7c61eca37bc5 not found
2021-09-10 12:49:00,934 - DEBUG - taskview:insert_existing_subtask:816 - Task 139a5dc9-4672-40e6-b9e3-6e14168d2665 not found
2021-09-10 12:49:00,934 - DEBUG - taskview:insert_existing_subtask:816 - Task 08d0bddb-3c7a-4328-8944-289e488fbfed not found
2021-09-10 12:49:00,934 - DEBUG - taskview:insert_existing_subtask:816 - Task 0743427c-1a29-42a6-a58d-4bcb7bbd1a54 not found
2021-09-10 12:49:00,934 - DEBUG - taskview:insert_existing_subtask:816 - Task edccfb7f-b420-4c1c-b0f9-c24dc5d3fc75 not found
2021-09-10 12:49:00,934 - DEBUG - taskview:insert_existing_subtask:816 - Task 1042d33b-40da-4b7d-ae8b-6984465839c9 not found
2021-09-10 12:49:00,934 - DEBUG - taskview:insert_existing_subtask:816 - Task 727f771c-5496-4736-9152-4b57527a8ecc not found
2021-09-10 12:49:00,934 - DEBUG - taskview:insert_existing_subtask:816 - Task 1781067e-3255-4812-8963-0ae94dce1c58 not found
2021-09-10 12:49:00,934 - DEBUG - taskview:insert_existing_subtask:816 - Task 286cbea8-04b4-4911-9fb3-d2a4f806e61e not found
2021-09-10 12:49:00,934 - DEBUG - taskview:insert_existing_subtask:816 - Task 333b03ff-81dc-451c-a6f3-8799c25adc22 not found
2021-09-10 12:49:00,934 - DEBUG - taskview:insert_existing_subtask:816 - Task 72cab2f3-f074-46d0-b571-07908d4050bb not found
2021-09-10 12:49:00,934 - DEBUG - taskview:insert_existing_subtask:816 - Task 52280ccc-505a-4c32-b5c4-4eb00130b903 not found
2021-09-10 12:49:00,934 - DEBUG - taskview:insert_existing_subtask:816 - Task 7dbb3eb1-17a6-408b-8ae7-6cfcc5827989 not found
2021-09-10 12:49:00,934 - DEBUG - taskview:insert_existing_subtask:816 - Task 25ee334d-4a1d-486d-984e-b79b5e854267 not found
2021-09-10 12:49:00,934 - DEBUG - taskview:insert_existing_subtask:816 - Task 457b2931-a15f-4a02-b6a8-656d9a0599db not found
2021-09-10 12:49:00,936 - DEBUG - taskview:process:175 - Processing text buffer after 250ms
2021-09-10 12:49:01,075 - DEBUG - taskview:process:225 - Processed in 139.08ms
2021-09-10 12:49:01,141 - DEBUG - taskview:process:175 - Processing text buffer after 250ms
2021-09-10 12:49:01,141 - DEBUG - taskview:process:225 - Processed in 0.66ms
2021-09-10 12:49:01,186 - DEBUG - taskview:process:175 - Processing text buffer after 250ms
2021-09-10 12:49:01,192 - DEBUG - taskview:process:225 - Processed in 6.35ms
```

**Comment by Neui** [comment]
Sounds like https://github.com/getting-things-gnome/gtg/issues/699

TL;DR: libpango which draws text crashes the application when drawing strike through text (~~this~~). When using flatpak, as a workaround you can downgrade `org.gnome.Platform` (and pin the version for now so you don't accidentally update it):

```sh
flatpak update --commit=f111d6c1e912823e8db0f6ee8eebff82f9a8794b444e85e3aa801fe10e7b8e59 org.gnome.Platform//40
flatpak mask org.gnome.Platform//40  # Pin current version so you don't accidentally update it
```
